### ISSUE TITLE
Get certificate paths task fails on RHEL 8 when run from Satellite 6.7

### ISSUE THREAD

**Issue by benformosa** [theforeman/ansible-foreman_scap_client#22]
When I run this task on a RHEL 8 host via the Ansible remote execution feature of Satellite 6.7, I get the following error:

```
TASK [theforeman.foreman_scap_client : Get certificate paths] ******************
fatal: [client.example.com]: FAILED! => {
    changed": false,
    "msg": "non-zero return code",
    "rc": 1,
    "stderr": "Failed to add the host to the list of known hosts (/usr/share/foreman-proxy/.ssh/known_hosts).\r\nConnection to client.example.com closed.\r\n",
    "stderr_lines": [
        "Failed to add the host to the list of known hosts (/usr/share/foreman-proxy/.ssh/known_hosts).",
        "Connection to client.example.com closed."
    ],
    "stdout": "\u001b[1mTraceback\u001b[m (most recent call last):\r\n\t1: from /home/satellite/.ansible/tmp/ansible-tmp-1606781345.0-26409-104605405404186/foreman_scap_client_facts.rb:27:in `<main>'\r\n/home/satellite/.ansible/tmp/ansible-tmp-1606781345.0-26409-104605405404186/foreman_scap_client_facts.rb:21:in `subscription_manager_cert_paths': \u001b[1mundefined method `+' for nil:NilClass (\u001b[1;4mNoMethodError\u001b[m\u001b[1m)\r\n\u001b[m",
    "stdout_lines": [
        "\u001b[1mTraceback\u001b[m (most recent call last):",
        "\t1: from /home/satellite/.ansible/tmp/ansible-tmp-1606781345.0-26409-104605405404186/foreman_scap_client_facts.rb:27:in `<main>'",
        "/home/satellite/.ansible/tmp/ansible-tmp-1606781345.0-26409-104605405404186/foreman_scap_client_facts.rb:21:in `subscription_manager_cert_paths': \u001b[1mundefined method `+' for nil:NilClass (\u001b[1;4mNoMethodError\u001b[m\u001b[1m)",
        "\u001b[m"
    ]
}
```

The `foreman_scap_client_facts.py` script seems to run OK when I run it manually on a RHEL 8 host.

The role also runs successfully when I run it with `ansible-playbook` from the Capsule server.

Could this be some issue with which python is used to run the script?
Would it be preferable to change the script to a module? (I'm working on a PR https://github.com/benformosa/ansible-foreman_scap_client/tree/fact_module)
Is there an existing Ansible module to read values from an ini config file?

**Comment by xprazak2** [comment]
Looking at the error, it seems to me like remote execution is having problem connecting to the target host. We have moved to a python script only [recently](https://github.com/theforeman/ansible-foreman_scap_client/commit/2f12d6e4a7151d20e0c7753790e56b93ba7c8e19), it replaced a ruby script that is shipped with 6.7. You can try the older version shipped with your Satellite to find out whether this is caused by python version or not.

**Comment by benformosa** [comment]
Hi, sorry about that, I hadn't realised Satellite still had the Ruby script.
I've tested with the existing python script and it works great.
Thanks for your help!
